cfgov/tccp/enums.py

```diff
@@ -156,67 +156,67 @@ def make_choices(*choices):
 )
 
 
-StateChoices = make_choices(
-    "AA",
-    "AE",
-    "AK",
-    "AL",
-    "AP",
-    "AR",
-    "AS",
-    "AZ",
-    "CA",
-    "CO",
-    "CT",
-    "DC",
-    "DE",
-    "FL",
-    "FM",
-    "GA",
-    "GU",
-    "HI",
-    "IA",
-    "ID",
-    "IL",
-    "IN",
-    "KS",
-    "KY",
-    "LA",
-    "MA",
-    "MD",
-    "ME",
-    "MH",
-    "MI",
-    "MN",
-    "MO",
-    "MP",
-    "MS",
-    "MT",
-    "NC",
-    "ND",
-    "NE",
-    "NH",
-    "NJ",
-    "NM",
-    "NV",
-    "NY",
-    "OH",
-    "OK",
-    "OR",
-    "PA",
-    "PR",
-    "PW",
-    "RI",
-    "SC",
-    "SD",
-    "TN",
-    "TX",
-    "UT",
-    "VA",
-    "VI",
-    "VT",
-    "WA",
-    "WI",
-    "WV",
-    "WY",
-)
+StateChoices = [
```

Comment:
@niqjohnson with this change you should be able to add these names to the details page as part of your work there, if you wish. Doing that would require a change like this:

```diff
diff --git a/cfgov/tccp/models.py b/cfgov/tccp/models.py
index 3eca4f5e4..a0e6fb3c1 100644
--- a/cfgov/tccp/models.py
+++ b/cfgov/tccp/models.py
@@ -407,6 +407,9 @@ class CardSurveyData(models.Model):
         Returns None if the card is available everywhere.
         """
         if self.availability_of_credit_card_plan == "One State/Territory":
-            return [self.state]
+            return [dict(enums.StateChoices)[self.state]]
         elif self.availability_of_credit_card_plan == "Regional":
-            return self.state_multiple
+            return [
+                dict(enums.StateChoices)[state]
+                for state in self.state_multiple
+            ]
```

I left that change out of this PR because I wasn't sure what you were thinking design-wise for how a long list might best appear on the details page.